posthog/queries/test/__snapshots__/test_retention.ambr

```diff
@@ -416,7 +416,7 @@
                         [
                           dateDiff(
                               'Day',
-                              toStartOfDay(toDateTime('2020-06-10 00:00:00', 'US/Pacific')),
+                              toStartOfDay(toDateTime('2020-06-10 07:00:00', 'US/Pacific')),
```

Comment:
Without digging in deeper, this seems like a bug?

Reply:
Since the regression to these lines happens in this PR, I'm also fine with the fix in here if it's a oneliner. I saw another "00:00" -> "16:00" in `test_lifecycle_query_runner.ambr`. If you think it's easier, then I have nothing against an additional PR either...

Reply:
Put into #19088